downstream/modules/platform/proc-controller-use-an-exec-env.adoc

```diff
@@ -0,0 +1,43 @@
+[id="proc-controller-use-an-exec-env"]
+
+= Using an execution environment in jobs
+
+.Prerequisites
+
+* An {ExecEnvShort} must have been created using ansible-builder as in xref:ref-controller-building-exec-env[Build an {ExecEnvShort}] 
+When an {ExecEnvShort} has been created, you can use it to run jobs. Use the {ControllerName} UI to specify the {ExecEnvShort} to use in your job templates.
+* Depending on whether an {ExecEnvShort} is made available for global use or tied to an organization, you must have the appropriate level of administrator privileges to use an {ExecEnvShort} in a job. 
+Execution environments tied to an organization require Organization administrators to be able to run jobs with those {ExecEnvShort}s.
+* Before running a job or job template that uses an {ExecEnvShort} that has a credential assigned to it, be sure that the credential contains a username, host, and password.
+
+To use an {ExecEnvShort} in a job, use the following procedure:
+
+.Procedure
+. On the navigation panel, select menu:Administration[Execution Environments].
+. Click btn:[Add] to add an {ExecEnvShort}.
+. Enter the appropriate details into the following fields:
+
+* *Name*: Enter a name for the {ExecEnvShort}(required).
+* *Image*: Enter the image name (required). 
+The image name requires its full location (repo), the registry, image name, and version tag in the example format of `quay.io/ansible/awx-ee:latestrepo/project/image-name:tag`.
```

Comment:
Outside the scope of this review, but I'm wondering whether we should be giving an example from somewhere other than quay.io?

Reply:
I wondered that, that's why I asked about searching Jiras